src/models/fields/IllnessStartDate.schema.ts

```diff
@@ -0,0 +1,50 @@
+import * as Joi from '@hapi/joi';
+
+const illnessStartDayErrorMessage: string = 'You must enter a day';
+const illnessStartMonthErrorMessage: string = 'You must enter a month';
+const illnessStartYearErrorMessage: string = 'You must enter a year';
+const invalidDate: string = 'Enter a real date';
+const invalidDateFuture: string = 'Start date must be today or in the past';
```

Comment:
you added to the variables of some of these constants "ErrorMessage" at the end of the name, but not with invalidDate and invalidDateFuture

```suggestion
const illnessStartDayErrorMessage: string = 'You must enter a day';
const illnessStartMonthErrorMessage: string = 'You must enter a month';
const illnessStartYearErrorMessage: string = 'You must enter a year';
const invalidDateErrorMessage: string = 'Enter a real date';
const invalidDateFutureErrorMessage: string = 'Start date must be today or in the past';
```